chi/clients.py

```diff
@@ -20,7 +20,7 @@
 
 session_factory = session
 
-NOVA_API_VERSION = "2.10"
+NOVA_API_VERSION = "2.61"
```

Comment:
Are we pinning api versions globally? what API calls need 2.61 instead of the default?

I think best practice is to override at the place in the client the API call is made, rather than globally